### PR TITLE
New testcases

### PR DESCRIPTION
**Pull request by gem-smitamishra**

### CODE REVIEW

**Comment by gem-smitamishra** [comment]
@gemAkshay Akshay please review latest change .

File: src/main/java/com/qa/mis/stepdefinition/AccountsPortalSteps.java [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+        } else {
+            GemTestReporter.addTestStep("element visibility", element.getText() + " is not present", STATUS.FAIL, DriverAction.takeSnapShot());
+        }
+    }
+
+    @Then("Click {string} sub-tab inside {string} tab")
+    public void clickSubTabInsideTab(String subtab, String tab) {
+        DriverAction.waitSec(5);
+        System.out.println(tab);
+        DriverAction.click(AccountsPortalLocator.sideTab(tab));
```
Comment (gemAkshay) [inline src/main/java/com/qa/mis/stepdefinition/AccountsPortalSteps.java:33]: step missing

File: src/main/java/com/qa/mis/stepdefinition/AccountsPortalSteps.java [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+            String parentWindow = DriverAction.getWindowHandle();
+            Set<String> S = DriverAction.getWindowHandles();
+            Iterator<String> I1 = S.iterator();
+            while (I1.hasNext()) {
+                String child_window = I1.next();
+                if (!parentWindow.equals(child_window)) {
+                    DriverAction.switchToWindow(child_window);
+                    String Title = DriverAction.getTitle(child_window);
+                    if (Title.equalsIgnoreCase(title)) {
+                        GemTestReporter.addTestStep("New Tab ", "Right tab is open", STATUS.PASS, DriverAction.takeSnapShot());
```
Comment (gemAkshay) [inline src/main/java/com/qa/mis/stepdefinition/AccountsPortalSteps.java:58]: Worth mentioning Tab name/Title here instead of "Right Tab"

**Review by gemAkshay — APPROVED** [review]
@gemini-akashGarg Approved

**Review by gemAkshay — APPROVED** [review]
@gemini-akashGarg Approved

**Review by gemAkshay — APPROVED** [review]
@gemini-akashGarg Approved

**Review by gemini-akashGarg — APPROVED** [review]

**Pull request closed without merging**